trunk/cli/README.md

```diff
@@ -1,51 +1,82 @@
-# Motivation
+# Trunk CLI
 
-In the course of building [CoreDB](https://coredb.io)'s Managed Postgres SaaS product, we've identified an exciting opportunity to help evolve the Postgres extension ecosystem. To date, the two most common hubs for Postgres extensions are Aptitude (`apt`) and the PostgreSQL Extension Network (`pgxn`). While these serve as important channels for publication and installation, they are not without criticism. One drawback to highlight is the apparent and necessary tradeoff between the registry's programmatic access and the amplified exposure the platform can offers its users. In order to synthesize these critical features, we're launching a novel, open-source home for Postgres extensions: `trunk`.
+The [Trunk PostgreSQL Extension Registry](https://pgtrunk.io), henceforth Trunk, offers a companion CLI (command line interface) to facilitate a user-friendly programmatic interface. This toolset, written in Rust, leverages the package manager, Cargo, and lowers the barriers to building, sharing, and using PostgreSQL extensions.
 
-# Introducing Trunk
+## Installation
 
-Trunk serves as registry where users can publish, search, and download community-made Postgres extensions. Inspired by popular developer hubs, such as [crates.io](http://crates.io) (Rust), [pypi.org](http://pypi.org) (Python), and [npmjs.com](http://npmjs.com) (JavaScript), Trunk aims to foster an information-rich environment. Here, developers can interact with the registry in a variety of ways and proudly showcase their contributions. Furthermore, users can gain insights into valuable metrics on extension downloads and trends.
+The Trunk CLI can be installed using the following command:
 
-At its core, the goal of trunk is to cultivate a thriving Postgres extension ecosystem by lowering the barriers to building, sharing, and using Postgres extensions.
+`cargo install pg-trunk`
 
-# Roadmap
+By default, the files are stored (). To confirm its proper installation, invoke the following:
 
-The Trunk infrastructure can be divided into the following: a command line interface (CLI), a registry, and a website.
+`trunk --version`
 
-### The CLI
+## Commands
 
 The CLI toolkit will abstract away many complexities in extension development and installation by using the following commands:
 
-`trunk init`
-- setup your environment to build a new Postrgres extension.
+- `trunk init` - setup your environment to build a new Postrgres extension.
```

Comment:
`trunk init` doesn't exist yet. Maybe we could have a section that highlights additional commands on the horizon.